app/uk/gov/hmrc/tariffclassificationfrontend/service/CasesService.scala

```diff
@@ -80,6 +80,14 @@ class CasesService @Inject()(appConfig: AppConfig,
     } yield updated
   }
 
+  def assignCase(original: Case, operator: Operator)
+                  (implicit hc: HeaderCarrier): Future[Case] = {
+    for {
+      updated <- connector.updateCase(original.copy(assignee = Some(operator)))
+      _ <- addAssignmentChangeEvent(original, updated, operator)
+      _ = auditService.auditCaseAssigned(updated, operator)
```

Comment:
do you mind changing the signature of `auditCaseAssigned` in `AuditService.scala`?
I think we should pass these arguments:
- Case c // the updated case
- Operator operator // the user that is logged in
- Operator assignee // the person that will work on that case

in this class `operator` will be the same as `assignee`, but at least we keep this definition more generic. In the future we will be able to have the assignee being different from the current logged in user.

Reply:
I think the signature is fine as it is, as you can get the assignee from `c.assignee`
I'll change it so the `assigneeId` in the audit payload is the actual `assignee` though rather than the operator who actioned the change

Reply:
Pushed